docs/kafka/topic-configuration-kafka/README.adoc

```diff
@@ -317,7 +315,7 @@ Message timestamp type determines whether the timestamp is generated when the me
 [cols="25%,75%"]
 |===
 h|Editable
-|No
+|Yes
```

Comment:
Is this change taking effect on non-trial Kafka Instances? On trial ones it's not Editable by default...

Reply:
@showuon Please confirm.

Reply:
This is not editable for standard Kafka instances either. Tested on stage.

Reply:
@mmusil Ack, changed back to No.